source/sdk/lang/Memory.ooc

```diff
@@ -1,53 +1,17 @@
-
-// as goofy as it sounds, memory-related routines are actually in string.h
-// yay C. (Also, Google Native Client doesn't have memory.h)
 include string
 version(windows) { include malloc }
```

Comment:
Removing the curly brackets here makes rock **always** try to import this file. I don't know if this is a known bug.